pkg/rules/rules.go

```diff
@@ -52,26 +65,80 @@ func Build(crs conf.Rules, clusters target.Clusters) (Rules, error) {
 	return rs, nil
 }
 
+// NewFromSlice builds new rules list from slice of rules. Used in tests
+func NewFromSlice(rules []Rule, ms *metrics.Prom) Rules {
```

Comment:
Not a showstopper. This is a shallow abstraction and is probably not worth it.

Reply:
Yes, I also find this a bit funny. :) 
Removed it.
Found a better way, if we provide config objects in tests, we can also make Compile private :)